docs/user-guide/cli-using-team-profiles-hierarchy.md

```diff
@@ -0,0 +1,111 @@
+
+## Using multiple properties found in multiple profiles
+
+Consider a more layered scenario. Again, assume you have all four configuration file types, but the following conditions apply:
+
+- There are multiple profiles across all four configuration file types.
+- Some profiles appear in multiple configurations. Other profiles show up in only one file.
+- There are multiple properties shared across several profiles.
+- Some properties are found in only one profile.
+
+In this scenario, the following profiles, properties, and values exist, displayed in the format **profile:** *property:* value:
+
+| Project User <br/> Configuration File |  Project Team <br/> Configuration File | Global User <br/> Configuration File |  Global Team <br/> Configuration File |
+| :----------------------  | :------------------------ | :---------------------- | :----------------------- |
+| **abc:** *direction:* north  | **abc:** *direction:* east      | **abc:** *direction:*  south | **abc:** *direction:* west  <br/> **abc:** *numbers*: 123    |
+| **def:** *shape:* triangle     | **def:** *shape:* square        | **def:** *shape:* circle     |                                |
+|                                |                                 |                              | **ghi:** *texture:* bumpy <br/>|
+| **jkl:** *temperature:* cold   |                                 |                              |                                |
+|                                | **mno:** *fruit:* banana        |                              |                                |
+|                                |                                 | **pqr:** *distance:* near    |                                |
+
+The following table shows how Zowe CLI determines which profiles, properties, and values to use in a command.
+
+| Configuration files in use &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;  &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; | Specificity hierarchy rules | Profiles, properties and values used &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;|
+| :--------- | :------------ |:------------ |
+| - global user profile <br/> - global team profile | - When the same property exists within the same profile in both config files, the property value from the global user config is used. <br/> - When the same profile exists in both config files, but a property of that profile exists in only one file, that property is used. <br/> - If a profile exists in only one config file, that profile is used in its entirety. | **abc:** *direction:* south <br/> **def:** *shape:* circle <br/> **ghi:** *texture:* bumpy <br/> **pqr:** *distance:* near |
```

Comment:
I think this should also include "**abc:** *numbers*: 123"

Reply:
added! thank you